UI/src/view/MassTransitStopLayer.js

```diff
@@ -354,14 +354,15 @@ window.MassTransitStopLayer = function(map, roadCollection, mapOverlay, assetGro
   };
 
   var createNewAsset = function(lonlat, placement) {
+    var directions_decode = {BothDirections: 2, TowardsDigitizing: 2, AgainstDigitizing: 3};
```

Comment:
I can't rename it, because it is used in line below (368) to decode the traffic direction.  I can add a comment before for example -> BothDirection varible takes the value 2, to assume the defaultAssetDirection (TowardsDigitizing)